packages/components/src/components/video/video.ts

```diff
@@ -0,0 +1,86 @@
+import { css, html } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { HasSlotController } from '../../internal/slot';
+import { property } from 'lit/decorators.js';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Used to wrap external video elements (e. g. from Moving Image or bare <video>-Tags) and provide some basic styling for Union Investment.
+ * @status experimental
+ * @since 1.0
+ *
+ * @dependency sd-example
+ *
+ * @event sd-play - Event emitted when the video is played or paused. Detail reflects "playing" property.
+ *
+ * @slot - The default slot used to pass a video player element.
+ * @slot play-icon - The video's play icon.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart play-button - Button element wrapper around the play-icon slot.
+ */
+@customElement('sd-video')
+export default class SdVideo extends SolidElement {
+  private readonly hasSlotController = new HasSlotController(this, '[default]', 'play-icon');
+
+  /** True if the contained video is playing. Hides everything when true. */
+  @property({ type: Boolean, reflect: true }) playing = false;
+
+  /** Show a darker overlay. */
+  @property({ type: Boolean, reflect: true }) overlay = false;
+
+  play() {
+    this.emit('sd-play');
+    this.playing = true;
+  }
```

Comment:
If we provide this, we should have a second method, that does the opposite.

Reply:
I don't think it makes sense to expose this or add a "pause" method, control is always coming from the wrapped component and should be reflected via the property.  

I propose marking this function as private and using only for the click event to make it simple.  External control should use the "playing" property exclusively.

Reply:
good point... @mariohamann what do you think?

Reply:
Ok. Either both or none. In accordion we provide it like this:

<img width="1050" alt="CleanShot 2023-10-11 at 13 23 20@2x" src="https://github.com/solid-design-system/solid/assets/26542182/cfa11ca3-e625-4270-b094-527d7995f532">

But I think it's totally valid to just work with the prop here.